src/docs/product/sentry-basics/tracing/trace-view.mdx

```diff
@@ -33,7 +33,8 @@ The transaction view provides a split view similar to the [span view](/product/s
 ### Orphan Traces and Broken Subtraces
 
 In a fully instrumented system, a span in one service will connect to a transaction in a subsequent service. For a variety of reasons, a transaction in a service may not arrive in Sentry. When Sentry encounters these types of transactions within a trace, we link the transactions with a dashed line since they can no longer be directly connected to the root.
-In addition, broken subtraces can occur when Sentry receives an error for a trace, but the corresponding transaction does not exist. In this case, the error will not appear on the trace view and the trace navigator won't appear on the **Issue Details** page. In these cases you can click "Open In Discover" to see all the events in the trace.
+In addition, broken subtraces can occur when Sentry receives an error for a trace, but the corresponding transaction does not exist. Such errors are linked using dashed lines and clicking on the row takes us to the corresponding **Issue Details** page.
```

Comment:
```suggestion
In addition, broken subtraces can occur when Sentry receives an error for a trace, but the corresponding transaction does not exist. Such errors are linked using dashed lines and clicking on the row takes you to the corresponding **Issue Details** page.
```

Since the rest of the page uses "you" not "us"